xbmc/rendering/dx/DeviceResources.cpp

```diff
@@ -1097,3 +1114,154 @@ void DX::DeviceResources::Trim() const
 }
 
 #endif
+
+void DX::DeviceResources::DetectDisplayHdrCapable(bool& hdr_capable, bool& hdr_enabled)
```

Comment:
Can we detect HDR capable display on ` DX::DeviceResources::HandleOutputChange`? if it possible it will be better to split this method for two - `bool IsDisplayHDRCapable()` and `bool IsDisplayHDREnabled()`.

Reply:
This would be nice but it penalizes performance because would have to call the function `pOutput6->GetDesc1` 2 times (and get the pointers).

But I will improve it so that the parameters are optional ("hdrCapable" is almost not used) and the function returns bool for "hdrEnabled".

Reply:
Done in https://github.com/xbmc/xbmc/pull/17012/commits/87d26b919c8c91a9ec7a0c74569d38d29c61067f